furms-core/src/main/java/io/imunity/furms/core/projects/ProjectServiceImpl.java

```diff
@@ -128,6 +128,16 @@ public void addMember(String communityId, String projectId, String userId){
 		usersDAO.addProjectMemberRole(communityId, projectId, userId);
 	}
 
+	@Override
+	@FurmsAuthorize(capability = PROJECT_LIMITED_WRITE, resourceType = PROJECT, id="projectId")
+	public void inviteMember(String communityId, String projectId, String email) {
+		Optional<User> user = usersDAO.findByEmail(email);
+		if (user.isEmpty()) {
+			throw new IllegalArgumentException("Could not invite user due to wrong email adress.");
```

Comment:
This is a general remark: instead of 'wrong' or 'invalid' it is more helpful to be more precise whenever possible. Here it is no problem to write 'empty'.

Reply:
I think empty instead of wrong will be less clear. It strongly suggest that emial is empty string, but it is empty optional.